Frontend/voitheia-App/src/app/app.module.ts

```diff
@@ -78,6 +79,10 @@ import { ImprintComponent } from './components/terms/imprint/imprint.component';
       provide: HTTP_INTERCEPTORS,
       useClass: TokenInterceptor,
       multi: true
+    },
+    {
+      provide: MAT_RADIO_DEFAULT_OPTIONS,
+      useValue: {color: 'primary'}
```

Comment:
We do not need it here since I am doing this in the custom Material module already

Reply:
deleted